src/View/PasswordQuickInput.ts

```diff
@@ -16,16 +16,22 @@
 
 import * as vscode from 'vscode';
 
-export async function showPasswordQuickInput(): Promise<string|undefined> {
+// NOTE ASCII characters have codes ranging from u+0000 to u+007f
```

Comment:
Could you move this code to the original code location? :)
Because it is hard to know which part has been modified.